tifeatures/factory.py

```diff
@@ -735,16 +738,16 @@ async def items(
                 )
 
             if offset:
-                query_params = dict(request.query_params)
-                query_params.pop("offset")
+                qp = dict(request.query_params)
```

Comment:
you don't like big variable names?

Reply:
so.... in my continuing love/hate relationship with mypy, for some reason mypy was typing query_params as QuerParams (ie request.query_params) and when I just changed the variable name out of exhasperation it worked fine :shrug:

Reply:
Hah, figured out why. I already was using the name query_params=QueryParams(...) in the block above.